src/routes/blogs.js

```diff
@@ -1,17 +1,18 @@
 import express from "express";
-import {param, validationResult} from "express-validator"
-
-import services from "../services"
+import { param, body, validationResult } from "express-validator";
 
+import services from "../services";
 
 const router = express.Router();
 
-router.get('/', (req, res) => {
-    res.send('We are on blogs');
-});
-
-router.post('/', (req, res) => {
-    console.log(req.body);
-});
+router.post(
+  "/",
+  body("title")
+    .isLength({ min: 3, max: 80 })
```

Comment:
do we have this validation in schema ? If not add it there, otherways it doesn't make sense